### PR TITLE
Fix scroll method setting invalid

### PR DESCRIPTION
**Pull request by jouyouyun**
Because of libinput only supported single method to set, so the scroll mode must be a single valid method.

### CODE REVIEW

File: src/xf86libinput.c [hunk 1]
```diff
@@ -3454,34 +3454,35 @@ LibinputSetPropertyScrollMethods(DeviceIntPtr dev,
...
-		modes |= LIBINPUT_CONFIG_SCROLL_2FG;
-	if (data[1])
-		modes |= LIBINPUT_CONFIG_SCROLL_EDGE;
-	if (data[2])
-		modes |= LIBINPUT_CONFIG_SCROLL_ON_BUTTON_DOWN;
+		mode |= LIBINPUT_CONFIG_SCROLL_2FG;
+	else if (data[1])
+		mode |= LIBINPUT_CONFIG_SCROLL_EDGE;
+	else if (data[2])
+		mode |= LIBINPUT_CONFIG_SCROLL_ON_BUTTON_DOWN;
```
Comment (whot) [inline src/xf86libinput.c]: This doesn't look right. The previous code forces the user to submit a correct property, if more than one bit is set we get a BadValue, as intended. With this bit now, I can set the property to 1 1 1 and it'll succeed, despite only one method being enabled.

Comment (jouyouyun) [inline src/xf86libinput.c]: Thank you for your reply,  I understand the meaning of the previous code.
 I hope you can change the description of the scroll method property, indicating that these methods are mutually exclusive.

Comment (whot) [inline src/xf86libinput.c]: it already says "only one is enabled at a time at max" but please feel free to reword this to make it more obvious. I wrote it, so it's already obvious to me ;)

Comment (jouyouyun) [inline src/xf86libinput.c]: Sorry, please forgive my carelessness, I have found it in the header file 'libinput-properties.h'.
But can it be added to the man manual? Thanks!
